kie-server-parent/kie-server-controller/kie-server-controller-api/src/main/java/org/kie/server/controller/api/model/events/ServerTemplateUpdated.java

```diff
@@ -15,17 +15,25 @@
 
 package org.kie.server.controller.api.model.events;
 
+import java.util.Objects;
+
 import org.kie.server.controller.api.model.spec.ServerTemplate;
 
 public class ServerTemplateUpdated implements KieServerControllerEvent {
 
     private ServerTemplate serverTemplate;
+    private boolean resetBeforeUpdate;
```

Comment:
@mswiderski It is a change to the Event Payload (aka ServerTemplateUpdated) rather than the actual ServerTemplate data model itself. Therefore, it will be no impact to current implementation of ServerTemplateConverter.

Reply:
it is also changed in server template https://github.com/kiegroup/droolsjbpm-integration/pull/1698/files#diff-9e4d13a201f4713a892ef4109229f719R46

Reply:
@mswiderski In such case, yes we will need to some change to accommodate this new field for ServerTemplateConverter. 
How do we coordinate this change? Since this PR (https://github.com/kiegroup/kie-wb-common/pull/2319) which includes ServerTemplateConverter is just get ready to be merged in few days. (Targeted to 7.3). Shall we deal with it in a followup PR after (AF-1685) being merged. Any other options?

Reply:
let's do it once this is merged, would you make a note of it?

Reply:
An issue (https://issues.jboss.org/browse/JBPM-8220) logged for this change.

Reply:
PR (https://github.com/kiegroup/droolsjbpm-integration/pull/1720) sent